client/settings/payment-method-settings/digital-wallets-settings.js

```diff
@@ -43,33 +72,58 @@ const buttonActionOptions = [
 		value: 'book',
 	},
 ];
+
+const makeButtonThemeText = ( string ) =>
+	interpolateComponents( {
+		mixedString: string,
+		components: {
+			br: <br />,
+			helpText: (
+				<span className="payment-method-settings__option-help-text" />
+			),
+		},
+	} );
 const buttonThemeOptions = [
 	{
-		label: __( 'Dark', 'woocommerce-payments' ),
+		label: makeButtonThemeText(
+			__(
+				'Dark {{br/}}{{helpText}}Recommended for white or light-colored backgrounds with high contrast.{{/helpText}}',
+				'woocommerce-payments'
+			)
+		),
 		value: 'dark',
 	},
 	{
-		label: __( 'Light', 'woocommerce-payments' ),
+		label: makeButtonThemeText(
+			__(
+				'Light {{br/}}{{helpText}}Recommended for dark or colored backgrounds with high contrast.{{/helpText}}',
+				'woocommerce-payments'
+			)
+		),
 		value: 'light',
 	},
 ];
 
 const DigitalWalletsSettings = () => {
-	const [ cta, setCta ] = useState( 'buy' );
-	const [ size, setSize ] = useState( 'default' );
-	const [ theme, setTheme ] = useState( 'dark' );
+	const [ actionType, setActionType ] = useDigitalWalletsButtonActionType();
+	const [ size, setSize ] = useDigitalWalletsButtonSize();
+	const [ theme, setTheme ] = useDigitalWalletsButtonTheme();
+
 	return (
 		<Card>
 			<CardBody>
 				<h4>{ __( 'Call to action', 'woocommerce-payments' ) }</h4>
 				<RadioControl
+					className="payment-method-settings__cta-selection"
+					label={ __( 'Call to action', 'woocommerce-payments' ) }
+					hideLabelFromVision
 					help={ __(
 						'Select a button label that fits best with the flow of purchase or payment experience on your store.',
 						'woocommerce-payments'
 					) }
-					selected={ cta }
+					selected={ actionType }
```

Comment:
Renaming to `buttonType`, since the DB setting is already called `payment_request_button_type` - at least that way it's closer to the original name 😄